src/components/UI/navbar/index.js

```diff
@@ -47,21 +47,23 @@ const Navbar = () => {
             setIsLoggedIn(false);
             throw new Error(`${response.status} (${response.statusText})`);
           }
+
           return response.json();
         })
         .then((responseJson) => {
-          if (!responseJson.incompleteUserDetails) {
-            setIsLoggedIn(true);
-            setUserData({
-              userName: responseJson.username,
-              firstName: responseJson.first_name,
-              profilePicture: responseJson.picture.url?.DEFAULT_AVATAR,
-            });
+          if (responseJson.incompleteUserDetails) {
```

Comment:
Is it a good idea to go  with `!!responseJson.incompleteUserDetails`?
In the contract we are going ahead with `incompleteUserDetails` as boolean only right?

Reply:
Yes @harshith-venkatesh incompleteUserDetails is a boolean property (see in below ss). Therefore the logic inside if condition will work perfectly. I didn't get , Did  you say `!!responseJson.incompleteUserDetails` in a better option than `responseJson.incompleteUserDetails`. If yes , how? Both checking the same thing right?
![image](https://user-images.githubusercontent.com/64394213/152151951-bbfc9db2-90e0-40d6-8efb-d54412033500.png)

Reply:
Since it is perfect Boolean i guess we are good here